tasks/prime_hhg_workflow.py

```diff
@@ -259,3 +178,14 @@ def _get_nested_object_from_mto(self, object_key, object_id):
 
         # If id was not provided select an object at random
         return random.choice(nested_array)
+
+    @property
+    def current_move(self):
+        headers = {"content-type": "application/json"}
+        resp = self.client.get(
+            prime_path(f"/move-task-orders/{self.current_move_id}"),
+            name=prime_path("/move-task-orders/{moveID}"),
+            headers=headers,
+            **self.user.cert_kwargs,
+        )
+        return resp.content
```

Comment:
Okay so I did find an issue here running `make load_test_prime_workflow`.  I was seeing failures in update_post_counseling because the move being returned from the prime_workflow get_stored was a json string instead of a python object data type.

I think here we want to call `resp.json()` I think to parse the response json similar to what we do elsewhere https://github.com/transcom/milmove_load_testing/blob/9d16a13572fbb6fb654181e190aa65754161b623/tasks/prime.py#L240

```sh
[2021-12-22 13:59:09,094] Duncans-MacBook-Pro.local/ERROR/locust.user.task: byte indices must be integers or slices, not str
Traceback (most recent call last):
  File "/Users/duncan/.local/share/virtualenvs/milmove_load_testing-7o6dLg4P/lib/python3.9/site-packages/locust/user/task.py", line 296, in run
    self.execute_next_task()
  File "/Users/duncan/.local/share/virtualenvs/milmove_load_testing-7o6dLg4P/lib/python3.9/site-packages/locust/user/task.py", line 321, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/Users/duncan/.local/share/virtualenvs/milmove_load_testing-7o6dLg4P/lib/python3.9/site-packages/locust/user/task.py", line 327, in execute_task
    task()
  File "/Users/duncan/workspace/milmove_load_testing/tasks/prime_hhg_workflow.py", line 63, in hhg_move
    super().update_post_counseling_information()
  File "/Users/duncan/workspace/milmove_load_testing/tasks/prime.py", line 584, in update_post_counseling_information
    move_task_order_id = move_task_order["id"]  # path parameter
TypeError: byte indices must be integers or slices, not str
```

https://docs.python-requests.org/en/master/user/quickstart/#json-response-content